.circleci/config.yml

```diff
@@ -12,6 +12,17 @@ jobs:
             source /usr/local/share/virtualenvs/tap-asana/bin/activate
             pip install -U 'pip<19.2' 'setuptools<51.0.0'
             pip install .[dev]
+      - run:
+          name: 'Unit Tests'
+          command: |
+            source /usr/local/share/virtualenvs/tap-asana/bin/activate
+            pip install nose coverage
+            nosetests --with-coverage --cover-erase --cover-package=tap_asana --cover-html-dir=htmlcov tests/unittests
+            coverage html
+      - store_test_results:
+          path: test_output/report.xml
+      - store_artifacts:
+          path: htmlcov
       - add_ssh_keys
```

Comment:
The `add_ssh_keys` is unnecessary. Also please update to the latest tap-tester image and use the tap_tester_sandbox environment variables over the sandbox ones.

Reply:
Removed `add_ssh_keys`. In PR #39, we have updated the tap tester image, sandbox, and the command to run the integration tests.